.gitignore

```diff
@@ -57,4 +57,5 @@ env.bak/
 venv.bak/
 
 # Doc distribution folder
-doc/dist
+docs/dist
+docs/api-dist
```

Comment:
Why do we have `api-dist` here if `build.py` writes both API and Ansible docs to the `dist` folder?

Reply:
done